### PR TITLE
Add copyright message on server start, clean up imports.

### PR DESCRIPTION
**Pull request by wearrrrr**
thanks for making me have to work on the 4th of july

### CODE REVIEW

**Comment by e9x** [comment]
THIS WON'T WORK, WE MESSAGE ON DISCORD.

File: src/createServer.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
-import { WebSocketServer } from "ws";
 import wisp from "./ConnectionHandler";
 import http from "node:http";
 import { Socket } from "node:net";
 
+console.clear();
+console.log("THIS SOFTWARE MUST NOT BE SOLD NEITHER ALONE NOR AS A PART OF A SOFTWARE PACKAGE");
```
Comment (ading2210) [inline src/createServer.ts:6]: You can't ban people from selling GPL software.

**Comment by ading2210** [comment]
The messages on startup are annoying without a way to disable it. If you're making a library, *never* hardcode console messages. Also, the messages themselves are misleading and don't really accomplish anything. It would be far better to simply state that this code is AGPL licensed and link to the source.

**Comment by ProgrammerIn-wonderland** [comment]
Yeah please don't console.log this, also it's not technically illegal to sell A/GPL software either

**Comment by e9x** [comment]
this is perfect :D

**Comment by genericness** [comment]
> this is perfect :D

:(

**Pull request closed without merging**